Change lpop_rpush_src_dest to work in batches

edgurgel opened the pull request:
It will fetch a maximum of 1000 at a time when enqueue jobs from the inprogress list

It closes https://github.com/edgurgel/verk/issues/35

keyan commented on lib/verk/workers_manager.ex:
```diff
@@ -90,8 +90,13 @@ defmodule Verk.WorkersManager do
   def handle_info(:enqueue_inprogress, state) do
-    :ok = QueueManager.enqueue_inprogress(state.queue_manager_name)
-    {:noreply, state, 0}
+    case QueueManager.enqueue_inprogress(state.queue_manager_name) do
```
Ah I see now what you meant in https://github.com/edgurgel/verk/pull/102/files#r92948579 👍

edgurgel commented:
Cool now I have pushed a change so that it says how many were added and how many left.

```

2017-04-09 16:43:03.160 [info] 2100 jobs still to be added to the queue default from inprogress list. Added 1000 jobs.

2017-04-09 16:43:03.163 [info] 1100 jobs still to be added to the queue default from inprogress list. Added 1000 jobs.

2017-04-09 16:43:03.167 [info] 100 jobs still to be added to the queue default from inprogress list. Added 1000 jobs.

2017-04-09 16:43:03.168 [info] No more jobs to be added to the queue default from inprogress list. Added 100 jobs.
```
